test/e2e/custom_route_test.go

```diff
@@ -104,6 +104,50 @@ func TestCustomRouterCerts(t *testing.T) {
 
 	err = checkRouteHostname(t, routeClient, "openshift-authentication", "oauth-openshift", "foo.bar.com")
 	require.NoError(t, err)
+
+	err = checkCustomSecretIsMountedOntoOauthDeployment(t, kubeClient)
+	require.NoError(t, err)
+}
+
+func checkCustomSecretIsMountedOntoOauthDeployment(t *testing.T, kubeClient *kubernetes.Clientset) error {
```

Comment:
you should check that your content is being served on the route instead

Reply:
Done.